avocado/utils/ssh.py

```diff
@@ -248,3 +248,28 @@ def copy_files(self, source, destination, recursive=False):
             return result.exit_status == 0
         except process.CmdError as exc:
             raise NWException("failed to copy file {}".format(exc))
+
+    def check_connection(self):
+        """
+        Check for ssh connection if failed remove control path file
+        and reconnect it.
+
+        :returns: True if success or else False
+        :rtype: bool
+        """
+        if not self.connect():
+            if not self.quit():
+                for root, dirct, files in os.walk("/root/.ssh"):
```

Comment:
We should make this not depending on root user. ~/.ssh for instance.

Reply:
try to use ~/.ssh this but won't get file path.